interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -817,6 +818,29 @@ public void computeEngineCreds(String serviceAccount, String oauthScope) throws
     assertEquals(goldenResponse, response);
   }
 
+  /** Test JWT-based auth. */
+  public void jwtTokenCreds(InputStream serviceAccountJson) throws Exception {
```

Comment:
According to the [Spec](https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md#jwt_token_creds):
```
received SimpleResponse.username is not empty and is in the json key file used by the auth library. The client can optionally check the username matches the email address in the key file.
```
You may want to pass in the json key and do the check.